src/Facebook/FacebookClient.php

```diff
@@ -162,7 +162,11 @@ public function getBaseGraphUrl($postToVideoUrl = false)
      */
     public function prepareRequestMessage(FacebookRequest $request)
     {
-        $postToVideoUrl = $request->containsVideoUploads();
+        $pathElements = explode('/', $request->getUrl());
```

Comment:
@YuanhuiChen  , only 'transfer' phase has video file attached. 'start' and 'finish' has not video file.

@SammyK , any suggestion for such cases?